docs/pubsub-usage.rst

```diff
@@ -116,41 +127,42 @@ Create a new pull subscription for a topic:
 
 .. doctest::
 
-   >>> from gcloud.pubsub import Topic
-   >>> from gcloud.pubsub import Subscription
-   >>> topic = Topic('topic_name')
-   >>> subscription = Subscription('subscription_name', topic)
+   >>> from gcloud import pubsub
+   >>> client = pubsub.Client()
+   >>> topic = client.topic('topic_name')
+   >>> subscription = pubsub.Subscription('subscription_name', topic)
```

Comment:
Yeah I felt that way when writing this. The only goal of this PR (with a very high number of commits) is to implement the client pattern.